### PR TITLE
Update for Packer

### PR DESCRIPTION
**Pull request by tjjh89017**
Using ig-json-parser, need to gradle clean then gradle build to gen
ig-json-parser

### CODE REVIEW

**Comment by tjjh89017** [comment]
@fntsrlike : I haven't remove origin `pack`, and rename origin `unpack` to `unpack2`
after you merge this, I will remove origin two unused method.

File: Application/src/main/java/io/github/antijava/marjio/network/StatusData.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+public class StatusData {
+
+    /*
+     * Player 0
+     * Block  1
+     * Item   2
+     */
+    public final int Player = 0;
+    public final int Block  = 1;
+    public final int Item   = 2;
```
Comment (fntsrlike) [inline Application/src/main/java/io/github/antijava/marjio/network/StatusData.java]: Fields never used
Why not enum? Or at least use `static`

U can refer https://github.com/NCU-Anti-Java/Marjio/blob/develop/Common/src/main/java/io/github/antijava/marjio/common/input/Event.java#L23

File: Application/src/test/java/io/github/antijava/marjio/network/PackerTest.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+ */
+public class PackerTest {
+
+    private StatusData player;
+    private StatusData block;
+    private StatusData item;
+
+    private final int PLAYER = 0;
+    private final int BLOCK = 1;
+    private final int ITEM = 2;
```
Comment (fntsrlike) [inline Application/src/test/java/io/github/antijava/marjio/network/PackerTest.java]: Change to use `StatusData.Player` if [final int](https://github.com/NCU-Anti-Java/Marjio/pull/48/files#r48528184) be `static`?

**Comment by fntsrlike** [comment]
FYI

``` java
// Bytes to String
String string = new String(byte[] data, int offset, int length);

// String to Bytes
byte[] data = string.getBytes();
```

**Comment by tjjh89017** [comment]
To Slove `StatusData__JSONHelper` you need to use Gradle clean & classes to gen that class

default

**Comment by tjjh89017** [comment]
I forgeot to set those member to `static`
I will fix it later

File: Application/src/test/java/io/github/antijava/marjio/network/PackerTest.java (hunk 2 above)
Comment (tjjh89017) [inline Application/src/test/java/io/github/antijava/marjio/network/PackerTest.java]: i'll fix it later

**Pull request merged**